### DISCUSSION TITLE
MacOS "rclone": executable file not found in $PATH

### DISCUSSION THREAD

**Discussion by dm9tr0** [garethgeorge/backrest#241] [Q&A] [answered]
I can successfully run restic to make backup to Gdrive via rclone config from terminal, but can not use rclone in backrest.
Repo: rclone:remote:Rclone/Restic
`Operation failed: [unknown] failed to init repo: init failed: command "/Users/<USER>/.local/share/backrest/restic-0.16.4 init --json -o sftp.args=-oBatchMode=yes" failed: exit status 1 Process STDOUT: Fatal: create repository at rclone:remote:Rclone/Restic failed: exec: "rclone": executable file not found in $PATH`

Also, don`t see options how to import existed repo.

Macos, backrest installed via brew.

**Comment by florian-str** [comment, marked as answer]
I'm not sure why backrest behaves different from the CLI version, but it seems they ship their own executable.

The issue can be fixed by adding `-o rclone.program=/path/to/rclone` as a flag in the repository settings.
See [this](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html#other-services-via-rclone) for context.

**Comment by dm9tr0** [comment]
Thanks a lot! Yes, it helps.
